buildstockbatch/eagle.py

```diff
@@ -494,7 +494,8 @@ def queue_post_processing(self, after_jobids=[], upload_only=False, hipri=False)
         # Configuration values
         account = self.cfg['eagle']['account']
         walltime = self.cfg['eagle'].get('postprocessing', {}).get('time', '1:30:00')
-
+        memory = self.cfg['eagle'].get('postprocessing', {}).get('memory', 85248)
```

Comment:
Is this right? I don't see a `memory` key in the postprocessing spec in the schema. Is this supposed to be

```suggestion
        memory = self.cfg['eagle'].get('postprocessing', {}).get('node_memory_mb', 85248)
```

Reply:
Good catch! Thanks. I had named it 'memory' at first but changed it to node_memory_mb to distinguish it from 'parquet_memory_mb'. Forgot to change it here, and it silently passed my test runs cause it just used the default.